docs/multi-cluster-mesh/active-passive/multi-cluster-active-passive.md

```diff
@@ -0,0 +1,106 @@
+# Single mesh multi cluster multiple networks with Istio operator
+
```

Comment:
I'd also link this: https://istio.io/latest/docs/setup/install/multicluster/primary-remote_multi-network/

Say that we are doing this automatcially and call out the naming difference, primary-remote vs active-passive.

Reply:
pls call out the naming difference: primary-remote vs active-passive